zebrad/tests/acceptance.rs

```diff
@@ -2606,3 +2606,83 @@ async fn state_format_test(
     }
     Ok(())
 }
+
+/// Test that fixed responses at zcashd for the `z_getsubtreesbyindex` method are the same as zebrad responses.
+///
+/// Require scyncronized zebrad cache state to query high heights.
+#[tokio::test]
+#[ignore]
+async fn rpc_z_getsubtreesbyindex_zcashd_test_vectors() -> Result<()> {
+    use serde_json::Value;
+
+    let _init_guard = zebra_test::init();
+
+    // We're only using cached Zebra state here, so this test type is the most similar
+    let test_type = TestType::UpdateZebraCachedStateWithRpc;
+    let network = Network::Mainnet;
+
+    let (mut zebrad, zebra_rpc_address) = if let Some(zebrad_and_address) =
+        spawn_zebrad_for_rpc(network, "fully_synced_rpc_test", test_type, true)?
+    {
+        tracing::info!("running fully synced zebrad RPC test");
+
+        zebrad_and_address
+    } else {
+        // Skip the test, we don't have the required cached state
+        return Ok(());
+    };
+
+    // Wait for zebrad to load the full cached blockchain, locally this can take up to 3 minutes.
+    std::thread::sleep(Duration::from_secs(180));
+
+    // Create an http client
+    let client = RpcRequestClient::new(zebra_rpc_address.expect(""));
+
+    // Create test vector matrix
+    let zcashd_test_vectors = vec![
+        (
+            "test_mainnet_sapling_0_1.json".to_string(),
+            "[\"sapling\", 0, 1]".to_string(),
```

Comment:
Optional: raw string literals will be easier to read here:
https://doc.rust-lang.org/rust-by-example/std/str.html#literals-and-escapes

This change can also be applied to Orchard.

```suggestion
            r#"["sapling", 0, 1]"#.to_string(),
```

Reply:
This doesn't work with my other suggestion below, so I'm deleting it.